cool-core/src/main/java/com/nus/cool/core/cohort/refactor/filter/RangeFilter.java

```diff
@@ -131,4 +132,14 @@ public void loadMetaInfo(MetaChunkRS metaChunkRS) {
     // for range Filter, no need to load info
   }
 
+  private static int convertPartToInt(String dataStr){
+    if (dataStr.contains("-")){
```

Comment:
?? this is very adhoc, the logic should be put in a class to handle time range properly

Reply:
Abstract the TimeDelimiter into RangeFilter